samples/app-anonymous-users/nodejs/appPackage/manifest.json

```diff
@@ -76,11 +76,11 @@
     "messageTeamMembers"
   ],
   "validDomains": [
-    "{{domain-name}}"
+    "${{BOT_DOMAIN}}"
   ],
   "webApplicationInfo": {
-    "id": "{{Microsoft-App-id}}",
-    "resource": "api://{{domain-name}}/botid-{{Microsoft-App-id}}"
+    "id": "${{AAD_APP_CLIENT_ID}}",
```

Comment:
Make sure that these place holders are updated in Readme for running the sample local.

Reply:
yes it is.